project/profiles/capacity4more/modules/c4m/user/c4m_user/c4m_user.module

```diff
@@ -285,7 +285,8 @@ function c4m_user_get_initials($name) {
 function c4m_user_name($user) {
   $wrapper = entity_metadata_wrapper('user', $user);
 
-  if ($user->uid == 0) {
+  // When an user is created, this function is called without uid.
+  if (!isset($user->uid) || $user->uid == 0) {
```

Comment:
This test here: https://github.com/capacity4dev/capacity4more/pull/1259/files#diff-f0eba0545c8687cd02f262be9a4bc054R28
Creates a new user to be removed afterwards. Without that change, the code crashes and the user is not created.